### PR TITLE
Sgds 226 form inputgrp

### PR DESCRIPTION
**Pull request by haphizi**
reopen pr

### CODE REVIEW

File: stories/InputGroup/InputGroup.stories.mdx [hunk 1]
```diff
@@ -322,104 +402,540 @@ Multiple add-ons are supported and can be mixed with checkbox and radio input ve
...
+          aria-describedby="select-addon2"
+        />
+    <Form.Select aria-label="Default select example">
+      <option selected value="1">Oneasdasdasdasdasdasdadas</option>
+      <option value="2">Two</option>
+      <option value="3">Three</option>
+    </Form.Select>
+  </InputGroup>
+  </Form>
+</Story>
```
[the remark this answers is not in the document]
Comment (rav3) [reply stories/InputGroup/InputGroup.stories.mdx:908]: Let the user decide on the width of the select element

File: package.json [hunk 2]
```diff
@@ -59,7 +59,7 @@
     "@testing-library/jest-dom": "^5.16.1",
     "@testing-library/react": "^12.1.2",
     "@types/enzyme": "^3.10.10",
-    "@types/react": "^17.0.37",
+    "@types/react": "^17.0.39",
     "@types/react-dom": "^17.0.11",
     "@types/react-transition-group": "^4.4.4",
     "@types/sinon": "^10.0.6",
```
Comment (clukhei) [inline package.json:65]: dont forget to remove formik and yup packages!

**Review by clukhei — COMMENTED** [review]
how come not exposing more props on argstable?

File: src/components/Nav.tsx [hunk 3]
```diff
@@ -0,0 +1,169 @@
+import classNames from 'classnames';
```
[the remark this answers is not in the document]
Comment (haphizi) [reply src/components/Nav.tsx:1]: i think it slipped in while merging between branches. will remove.

**Review by clukhei — APPROVED** [review]

**Review by rav3 — APPROVED** [review]

**Pull request merged**